File: oi_china/app/src/main/java/com/example/richard/oichina/MainActivity.java
package com.example.richard.oichina;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void licoes(View view) {
        Intent intent = new Intent(this, Lessons.class);
        startActivity(intent);
    }
}

File: oi_china/app/src/main/java/com/example/richard/oichina/activity/MainActivity.kt
package com.example.richard.oichina.activity

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.widget.Button
import com.example.richard.oichina.R
import kotlinx.android.synthetic.main.activity_main.*


class MainActivity : AppCompatActivity() {

    private lateinit var btnLicoes: Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        setUpVariables()
        setUpClickHandle()
    }

    private fun setUpVariables() {
        btnLicoes = btn_licoes
    }

    private fun setUpClickHandle() {
        btnLicoes.setOnClickListener {
            val intent = Intent(this@MainActivity, LessonsActivity::class.java)
            startActivity(intent)
        }
    }
}